user/reference/overview.md

```diff
@@ -38,23 +31,21 @@ A [Windows](/user/reference/windows/) environment running Windows Server 2016.
-| Approx boot time     | 20-50s                                | 1-6s                              | 20-50s                               | 60-90s                       | 60-120s                            |
-| File system          | EXT4                                  | devicemapper on XFS               | EXT4                                 | HFS+                         | NTFS                               |
-| Operating system     | Ubuntu 12.04                          | Ubuntu 14.04                      | Ubuntu 14.04                         | OS X                         | Windows Server 2016                |
-| Memory               | 7.5 GB                                | 4 GB max                          | 7.5 GB                               | 4 GB                         | 8 GB                               |
-| Cores                | 2                                     | 2                                 | 2                                    | 2                            | 2                                  |
-| IPv4 network         | IPv4 is available                     | IPv4 is available                 | IPv4 is available                    | IPv4 is available            | IPv4 is available                  |
-| IPv6 network         | IPv6 is not available                 | IPv6 is available on loopback     | IPv6 is not available                | IPv6 is not available        | IPv6 is not available              |
-| Available disk space | approx 22GB                           | approx 9GB                        | approx 18GB                          | approx 41GB                  | approx 19 GB                       |
-
-> Note that the following keywords in your .travis.yml file will silently route your builds to our sudo-enabled infrastructure (even if you specify `sudo: false`): `docker`, `ping`, `sudo`.
+|                      | [Ubuntu Precise](/user/reference/precise) | [Ubuntu Trusty](/user/reference/trusty) | [OS X](/user/reference/osx/) | [Windows](/user/reference/windows) |
+|:---------------------|:------------------------------------------|:----------------------------------------|:-----------------------------|:-----------------------------------|
+| Name                 | Precise                                   | Ubuntu                                  | OS X                         | Windows                            |
+| Status               | Current                                   | Current                                 | Current                      | Early release                      |
+| Infrastructure       | Virtual machine on GCE                    | Virtual machine on GCE                  | Virtual machine              | Virtual machine on GCE             |
+| `.travis.yml`        | `dist: precise`                           | `dist: trusty`                          | `os: osx`                    | `os: windows`                      |
```

Comment:
do we want to include this?:
```
os: linux
dist: trusty || dist: precise
```
it might help imply how you can matrix:
```
os:
  - linux
  - osx
  - windows
```

Reply:
Yeah, we discussed unifying the two columns infact, so the table becomes an OS table of Linux / OSX / Win -> which makes more sense now.